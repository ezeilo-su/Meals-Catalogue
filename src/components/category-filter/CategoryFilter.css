.category-filter {
  width: 30%;
  margin: 30px 15px;
  display: flex;
  align-items: center;
  justify-content: space-between;
}

.category-filter h3 {
  color: #fff;
  font-size: 16px;
  height: 30px;
  text-align: center;
  line-height: 30px;
  margin: 0;
  padding: 0 20px;
  border-radius: 0 10px;
  background-color: #3aa;
}

.category-filter select {
  width: 150px;
  height: 30px;
  outline: none;
  margin: 0;
  border-radius: 10px;
}
