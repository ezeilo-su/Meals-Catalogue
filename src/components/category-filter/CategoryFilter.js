import React from 'react';

const CategoryFilter = () => (
  <div>
    <p>Filter</p>
  </div>
);

export default CategoryFilter;
