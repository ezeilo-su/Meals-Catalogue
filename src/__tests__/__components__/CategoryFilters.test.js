import React from 'react';
import renderer from 'react-test-renderer';
import CategoryFilter from '../../components/category-filter/CategoryFilter';

describe('CategoryFilter', () => {
  const allCat = ['cat1', 'cat2'];
  const handleFilterChange = () => {};

  test('Componenet renders correctly', () => {
    const component = renderer.create(
      <CategoryFilter onFilterChange={handleFilterChange} categoryList={allCat} />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
