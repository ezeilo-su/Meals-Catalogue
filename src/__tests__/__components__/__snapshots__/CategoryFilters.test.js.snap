// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryFilter Componenet renders correctly 1`] = `
<div
  className="category-filter"
>
  <h3>
    <span>
      Select category
    </span>
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M15.5 5H11l5 7-5 7h4.5l5-7z"
      />
      <path
        d="M8.5 5H4l5 7-5 7h4.5l5-7z"
      />
    </svg>
  </h3>
  <select
    defaultValue="All"
    name="allMealCategories"
    onChange={[Function]}
  >
    <option
      value="All"
    >
      All
    </option>
    <option
      value="cat1"
    >
      cat1
    </option>
    <option
      value="cat2"
    >
      cat2
    </option>
  </select>
</div>
`;
